feature-libs/product-configurator/rulebased/components/service/configurator-storefront-utils.service.ts

```diff
@@ -16,6 +16,7 @@ export class ConfiguratorStorefrontUtilsService {
   constructor(
     protected configuratorGroupsService: ConfiguratorGroupsService,
     @Inject(PLATFORM_ID) protected platformId: any,
+    @Inject(DOCUMENT) protected document,
```

Comment:
If changes needed: We need to have schematics for that

Reply:
Done!